.github/license-lint-config.yml

```diff
@@ -0,0 +1,68 @@
+unrestricted_licenses:
```

Comment:
these are all the cncf approved licenses that gatekeeper uses today (directly or indirectly, including for test file)

Reply:
Are there any CNCF-approved licenses that Gatekeeper doesn't use? They should be in this list if they are CNCF-approved